TM1py/Services/ElementService.py

```diff
@@ -67,6 +67,12 @@ def exists(self, dimension_name: str, hierarchy_name: str, element_name: str, **
             element_name)
         return self._exists(url, **kwargs)
 
+    def update_or_create(self, dimension_name: str, hierarchy_name: str, element: Element, **kwargs) -> Response:
+        if self.exists(dimension_name=dimension_name, hierarchy_name=hierarchy_name, element=element.name, **kwargs):
```

Comment:
The function exists requires the argument `element_name`

TypeError: ElementService.exists() missing 1 required positional argument: 'element_name'

Reply:
fixed exists function argument bug